mkt/constants/regions.py

```diff
@@ -108,7 +125,7 @@ class RESTOFWORLD(REGION):
     if country.get('ratingsbody'):
         country['ratingsbody'] = getattr(ratingsbodies, country['ratingsbody'])
 
-    globals()[k] = type(k, (REGION,), country)
+    locals()[k] = type(k, (REGION,), country)
```

Comment:
Does this have any bad effects? I thought it might have been `locals` and then it was later changed to `globals` for some reason? I may be misremembering.